### PR TITLE
Get conformance tests and test images from knative.dev/networking

### PR DESCRIPTION
**Pull request by tcnghia**
Once this is done I will follow up with may be moving `config-network.yaml` to knative.dev/networking as well.

### CODE REVIEW

**Comment by tcnghia** [comment]
PR https://github.com/knative/networking/pull/57 is staged to move `config-network.yaml` into `knative.dev/networking`

File: go.mod [hunk 1]
```diff
@@ -10,9 +10,8 @@ require (
 	k8s.io/api v0.18.1
 	k8s.io/apimachinery v0.18.6
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
-	knative.dev/networking v0.0.0-20200730222100-e8df1ba775fa
+	knative.dev/networking v0.0.0-20200801021400-4b262113649d
 	knative.dev/pkg v0.0.0-20200731005101-694087017879
-	knative.dev/serving v0.16.1-0.20200731105301-2da382612fb7
```
Comment (mattmoor) [inline go.mod:15]: 🤩

**Pull request merged**